packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator/perf/PluginInitializer.cs

```diff
@@ -11,7 +11,7 @@ internal static class PluginInitializer
         public static void Initialize()
         {
             PluginHandler pluginHandler = new PluginHandler();
-            pluginHandler.LoadPlugin(new CommandLineOptions
+            pluginHandler.LoadPlugin(null!, new CommandLineOptions
```

Comment:
We shouldn't need to do this. We should either pass in the emitter or remove the parameter.